cms/envs/aws.py

```diff
@@ -57,8 +57,7 @@
 BROKER_POOL_LIMIT = 0
 BROKER_CONNECTION_TIMEOUT = 1
 
-# For the Result Store, use the django cache named 'celery'
-CELERY_RESULT_BACKEND = 'djcelery.backends.cache:CacheBackend'
+CELERY_RESULT_BACKEND = 'cache+memcached://127.0.0.1:11211/'
```

Comment:
This should be configurable and should be able to handle a list of memcache nodes as input if it can't reuse a named cache provided by the Django cache framework.